dockstore-integration-testing/src/test/java/io/dockstore/client/cli/CheckUrlHelperFullIT.java

```diff
@@ -16,87 +16,99 @@
 
 package io.dockstore.client.cli;
 
-import static io.dockstore.client.cli.BaseIT.BIOWORKFLOW;
-import static io.dockstore.client.cli.BaseIT.getWebClient;
+import static io.dockstore.client.cli.BaseIT.getOpenAPIWebClient;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import io.dockstore.common.CommonTestUtilities;
 import io.dockstore.common.ConfidentialTest;
 import io.dockstore.common.TestingPostgres;
+import io.dockstore.openapi.client.ApiClient;
+import io.dockstore.openapi.client.api.HostedApi;
+import io.dockstore.openapi.client.api.WorkflowsApi;
+import io.dockstore.openapi.client.model.Workflow;
+import io.dockstore.openapi.client.model.WorkflowSubClass;
+import io.dockstore.openapi.client.model.WorkflowVersion;
 import io.dockstore.webservice.DockstoreWebserviceApplication;
-import io.dockstore.webservice.DockstoreWebserviceConfiguration;
 import io.dropwizard.testing.ConfigOverride;
-import io.dropwizard.testing.junit.DropwizardAppRule;
-import io.swagger.client.ApiClient;
-import io.swagger.client.api.WorkflowsApi;
-import io.swagger.client.model.Workflow;
-import io.swagger.client.model.WorkflowVersion;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
 import java.util.Optional;
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
-import org.junit.rules.TestRule;
-import org.junit.rules.TestWatcher;
-import org.junit.runner.Description;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
-@Category(ConfidentialTest.class)
 @Tag(ConfidentialTest.NAME)
+@ExtendWith(DropwizardExtensionsSupport.class)
```

Comment:
Since this class doesn't extend any of the base ITs, might be good to add the 
```
@ExtendWith(MuteForSuccessfulTests.class)
@ExtendWith(TestStatus.class)
```
annotations so we get that functionality.